mandatory/3-function-output.js

```diff
@@ -1,55 +1,58 @@
-// Add comments to explain what this function does. You're meant to use Google!
-function getNumber() {
-  return Math.random() * 10;
-}
-
-// Add comments to explain what this function does. You're meant to use Google!
-function s(w1, w2) {
-  return w1.concat(w2);
-}
-
-function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
-}
-
-/* 
-===================================================
-======= TESTS - DO NOT MODIFY BELOW THIS LINE =====
-
-There are some Tests in this file that will help you work out if your code is working.
-
-To run these tests type `node 3-function-output` into your terminal
-==================================
-*/
-
-const util = require("util");
-
-function test(test_name, actual, expected) {
-  let status;
-  if (actual === expected) {
-    status = "PASSED";
-  } else {
-    status = `FAILED: expected: ${util.inspect(
-      expected
-    )} but your function returned: ${util.inspect(actual)}`;
-  }
-
-  console.log(`${test_name}: ${status}`);
-}
-
-test(
-  "concatenate function - case 1 works",
-  concatenate("code", "your", "future"),
-  "code your future"
-);
-test(
-  "concatenate function - case 2 works",
-  concatenate("I", "like", "pizza"),
-  "I like pizza"
-);
-test(
-  "concatenate function - case 3 works",
-  concatenate("I", "am", 13),
-  "I am 13"
-);
+// Add comments to explain what this function does. You're meant to use Google!
+// This function get a random number between 0 and 1 inclusive 0 but not 1 multiply by 10
+function getNumber() {
+  return Math.random() * 10;

```

Comment:
Hey there great stuff on the previous work **however**....

- So when you use ```Math.random() * 10``` this won't really guarantee giving you **whole** number 1-10 (Note the question was **ambiguous** as to are floating number required but if for instance **only whole numbers were needed**) this would not generate one **from what I know I stand to be corrected**

- To generate random whole number from 0-10  code would look like this below:
```js
Math.floor(Math.random() * 10)
```

Reply:
Hi Ntshembo thank you for your review I will go back and make the necessary changes.